### ISSUE TITLE
 GO:0007189  adenylate cyclase-activating G-protein coupled receptor signaling pathway... missing parent

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#13903]
GO:0007189 adenylate cyclase-activating G-protein coupled receptor signaling pathway should be 
is_a cAMP mediated signalling?

**Comment by ukemi** [comment]
From the dfinitions, it looks like the cAMP-mediated signaling is downstream of the receptor signaling pathway. So the signaling pathway has_part positive regulation of cAMP-mediated signaling already.

**Comment by ValWood** [comment]
I think it is part of  the pathway.
Otherwise the pathway would only be the first 2 steps?
The pathway is (see the image Antonia posted)
https://github.com/pombase/curation/issues/1582

the def is
The series of molecular signals generated as a consequence of a G-protein coupled receptor binding to its physiological ligand, where the pathway proceeds through activation or inhibition of adenylyl cyclase activity and a subsequent change in the concentration of cyclic AMP (cAMP).

 the comment clarifies it should be used for upstream and downstream of cAMP, which makes sense to me? (the def is not clear though, otherwise the comment would not be required....)


@Antonialock

**Comment by ValWood** [comment]
Comment | This term is intended to cover steps in a GPCR signaling pathway both upstream and downstream of adenylate-cyclase activity. For steps upstream of adenylate cyclase activity, consider instead annotating to 'regulation of adenylate cyclase activity involved in G-protein coupled receptor signaling pathway ; GO:0010578.
-- | --

Although i wouldn't so the second part of the comment.

I would annotate the receptor with MF "adenylate cyclase activator activity" instead 
https://github.com/pombase/curation/issues/1610
(that is the only step upstream of adenylate cyclase).

**Comment by ValWood** [comment]
perhaps you were looking at this def
cAMP-mediated signaling

Definition | Any intracellular signal transduction in which the signal is passed on within the cell via cyclic AMP (cAMP). Includes production of cAMP, and downstream effectors that further transmit the signal within the cell.

i would still say that this does not exclude
GO:0007189 adenylate cyclase-activating G-protein coupled receptor signaling pathway

*because* the first 2 steps in 
GO:0007189 adenylate cyclase-activating G-protein coupled receptor signaling pathway
are involved in the production of cAMP

GO:0007189 adenylate cyclase-activating G-protein coupled receptor signaling pathway
is-a specific cAMP signalling pathway

**Comment by ValWood** [comment]
See abstract
https://www.ncbi.nlm.nih.gov/pubmed/24297439
https://www.ncbi.nlm.nih.gov/pubmed/18430926
https://www.ncbi.nlm.nih.gov/pubmed/16489217
https://www.ncbi.nlm.nih.gov/pubmed/16143612
https://www.ncbi.nlm.nih.gov/pubmed/15831585

**Comment by ValWood** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/15667320
review
The fission yeast Schizosaccharomyces pombe primarily detects glucose via a cAMP-signalling pathway. Components of this pathway include the Git3 G-protein-coupled receptor and a heterotrimeric G-protein, from which the Gpa2 Galpha subunit activates adenylate cyclase (Git2/Cyr1). Three additional proteins, Git1, Git7 and Git10 are required to generate a cAMP response even in a strain expressing an activated form of Gpa2, which is capable of bypassing the loss of the GPCR and Gbetagamma dimer.......

**Comment by ValWood** [comment]
Enough?

**Comment by ValWood** [comment]
From Charlie Hoffman:

I would agree with that, however I would say that the following is somewhat different.

GO:0043950 positive regulation of cAMP-mediated signaling


For example, hsp90 and git7 are positive regulators, but are not really part of the G protein pathway.  They are required to assemble Git1 and adenylyl cyclase into a complex.  Git1 may also be more appropriate to this designation as it is required for cyclase activation by Gpa2, but is not really part of the G protein signaling pathway.

In contrast, Git3, Gpa2, Git5, and Git11 are all squarely in the "GO:0010619 - adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway" category

(so the rest is something for us to sort out, we need to remove quite a few indirect phenotypes from git7 and hsp90 but  GO:0007189 is_a cAMP-mediated signalling pathway)

**Comment by ukemi** [comment]
OK. But a group should look at this globally as part of the signaling workshop. Does this close the other ticket too?

**Comment by ValWood** [comment]
Both can close.

I will make a proposal for the workshop to get rid of some of the MF in BP  terms like:

GO:0007190 activation of adenylate cyclase activity 
 that we should not need to use if we curate the MF steps correctly

**Comment by ukemi** [comment]
Yes, but we still need people to claim this pathway if we are going to address it at the workshop.

**Comment by ukemi** [comment]
Reopend so that this can be examined in the context of the signaling pathway work.

**Comment by ValWood** [comment]
@pgaudet  I think this is basically the modularization issue you mentioned on the GO annotation call a while back.

**Comment by ukemi** [comment]
@pgaudet  is this done?

**Comment by pgaudet** [comment]
I think we say it's resolved. 

@ValWood @Antonialock Does that look OK to you now ? 

<img width="684" alt="image" src="https://user-images.githubusercontent.com/4782928/49293106-54905780-f4af-11e8-801d-553438c12923.png">

**Comment by ValWood** [comment]
yup.

**Comment by pfey03** [comment]
Thanks @ValWood  for linking above issue, sorry being so late to this: 
GO:0007189 adenylate cyclase-activating G protein-coupled receptor signaling pathway, is still a parent to the dicty specific GO:0140582 'adenylate cyclase-activating G protein-coupled cAMP receptor signaling pathway'
and cannot be a child of 'cAMP signaling pathway'. So far it looks fine I hope it stays so. https://www.ebi.ac.uk/QuickGO/term/GO:0007189
